04/index.html

```diff
@@ -13,59 +13,59 @@
     <div class="navbar">
         <a href="#home">Home</a>
 
-        <div class="subnav">
+        <nav class="subnav">
             <a href="#items" class="subnavbtn">Items</a>
-            <div class="subnav-content">
-                <div class="subnav-item">
+            <ul class="subnav-content">
+                <li class="subnav-item">
                     <a href="#item1">item1</a>
-                    <div class="secline-subnav-content">
+                    <ul class="secline-subnav-content">
                         <a href="#item1.1">item1.1</a>
-                    </div>
-                </div>
-                <div class="subnav-item">
+                    </ul>
+                </li>
+                <li class="subnav-item">
                     <a href="#item2">item2</a>
-                    <div class="secline-subnav-content">
+                    <ul class="secline-subnav-content">
                         <a href="#item2.1">item2.1</a>
-                    </div>
-                </div>
-            </div>
-        </div>
+                    </ul>
+                </li>
+            </ul>
+        </nav>
 
-        <div class="subnav">
+        <nav class="subnav">
             <a href="#settings" class="subnavbtn">Settings</a>
-            <div class="subnav-content">
-                <div class="subnav-item">
+            <ul class="subnav-content">
+                <li class="subnav-item">
                     <a href="#option1">option1</a>
-                    <div class="secline-subnav-content">
+                    <ul class="secline-subnav-content">
                         <a href="#option1.1">Option1.1</a>
-                    </div>
-                </div>
-                <div class="subnav-item">
+                    </ul>
+                </li>
+                <li class="subnav-item">
                     <a href="#option2">option2</a>
-                    <div class="secline-subnav-content">
+                    <ul class="secline-subnav-content">
                         <a href="#option2.1">Option2.1</a>
-                    </div>
-                </div>
-            </div>
-        </div>
+                    </ul>
+                </li>
+            </ul>
+        </nav>
 
-        <div class="subnav">
+        <nav class="subnav">
             <a href="#contact" class="subnavbtn">Contact</a>
-            <div class="subnav-content">
-                <div class="subnav-item">
+            <ul class="subnav-content">
```

Comment:
Uważaj na drobne odstępu z prawej strony, jak przejedziesz wolno kursorem to stracisz "hover" i nie najdziesz na podmenu.